PWGHF/Utils/utilsEvSelHf.h

```diff
@@ -181,8 +197,9 @@ struct HfEventSelection : o2::framework::ConfigurableGroup {
   static constexpr char NameHistPosYAfterEvSel[] = "hPosYAfterEvSel";
   static constexpr char NameHistNumPvContributorsAfterSel[] = "hNumPvContributorsAfterSel";
   static constexpr char NameHistCollisionsCentOcc[] = "hCollisionsCentOcc";
+  static constexpr char NameHistUPC[] = "hUPCollisions";
```

Comment:
```suggestion
  static constexpr char NameHistUpCollisions[] = "hUpCollisions";
```
Keep the correspondence.